Sources/Views/Account/Account.swift

```diff
@@ -548,9 +548,9 @@ private func subscriptionTeammateOverview(_ data: AccountData) -> Node {
     enterpriseShareLink = []
   }
 
-  return .gridRow(
+  return Node.gridRow(
```

Comment:
Without making these `Node`s explicit these expressions took over a minute to compile.